lifecycle-operator/apis/lifecycle/v1alpha1/common/common.go

```diff
@@ -1,3 +1,5 @@
+// +groupName=lifecycle.keptn.sh
```

Comment:
why are those needed now?

Reply:
because otherwise the underlying types are not autogenerated correctly see https://github.com/elastic/crd-ref-docs/issues/58